src/EventListener/DeserializeListener.php

```diff
@@ -113,7 +114,7 @@ private function getFormat(Request $request): string
          */
         $contentType = $request->headers->get('CONTENT_TYPE');
         if (null === $contentType) {
-            throw new NotAcceptableHttpException('The "Content-Type" header must exist.');
+            throw new UnsupportedMediaTypeHttpException('The "Content-Type" header must exist.');
```

Comment:
https://github.com/api-platform/core/pull/2892/files#diff-0170cc702a42034d94c5d83ccadad1deR108 needs to be updated too.

Reply:
The link looks bad. Can you edit?

Reply:
The phpdoc:

```diff
-                  * @throws NotAcceptableHttpException
+                  * @throws UnsupportedMediaTypeHttpException
```

Reply:
got it, thanks